x-pack/filebeat/input/entityanalytics/provider/azuread/authenticator/oauth2/oauth2_test.go

```diff
@@ -62,12 +63,13 @@ func TestRenew(t *testing.T) {
 		value := "test-value"
 		expiresIn := 1000
 
-		srv := testSetupServer(t, value, expiresIn)
+		clientSecret := "value&chars=to|escape"
```

Comment:
I think I would have just changed the name of the label here. From the [docs](https://securego.io/docs/rules/g101):

> The use of hard-coded passwords increases the possibility of password guessing tremendously. This plugin test looks for all string literals and checks the following conditions:
>
> Variables are considered to look like a password if they have match any one of:
>
> - “password”
> - “pass”
> - “passwd”
> - “pwd”
> - “secret”
> - “token”

The next line is telling, "Note: this can be noisy and may generate false positives." This linter rule should probably not exist.